paddle/fluid/platform/mkldnn_reuse.h

```diff
@@ -989,6 +989,11 @@ class ActivationMKLDNNHandler
       // else
       //   out = scale*(X + bias) = scale*X + scale*bias
       if (!bias_after_scale) beta *= alpha;
+    } else if (ctx.Type() == "clip") {
```

Comment:
I'd suggest 
`} else if (ctx.Type() == "clip" || ctx.Type() == "clip_grad") {`
then you can get rid of if block 1038:1044

Reply:
Unfortunately I cannot do that because `ctx.Type() == "clip"` and `ctx.Type() == "clip_grad"` are in different functions(first in forward handler, second one in backward handler)